samples/skills/QASkill/MemoryQuery/skprompt.txt

```diff
@@ -0,0 +1,7 @@
+Someone wants to know the following about a GitHub repository:
```

Comment:
nit: Feels like this should either be a GitHub Skill or at least GitHub specific in the function naming. Perhaps this should just be part of the GitHubSkillsExample.csproj? Yeah, it looks like that project is already defining a skill and functions. Can we put this there?

Reply:
agreed, the function name is generic but the function content is specific to GitHub. I would rename the function to `GitHubQuery` or `GitHubMemoryQuery` or similar

Reply:
This was renamed (I don't own the PR so can't resolve)